src/components/PaymentsContainer.tsx

```diff
@@ -0,0 +1,36 @@
+import react from "react";
+import Image from "next/image";
+
+export default function PayMentsContainer() {
+  return (
+    <div className="bg-background-payments-colour flex items-center flex-col w-full">
+      <div className="text-black flex justify-center text-2xl font-sans">
+        Safe, easy payment options
+      </div>
+      <ul className="flex justify-center flex-row items-center">
+        {PaymentIcons.map((icon, index) => (
+          <Image className="flex items-center m-10" src={icon.icon} height={icon.size[1]} width={icon.size[0]} alt="test" key={index} />
+        ))}
+      </ul>
```

Comment:
Why is ul being used with li? Seems odd an unordered list isn't using list elements.

Another thing to note is that the icon.size[0] and icon.size[1] seems like it's been poorly designed. Why isn't it potentially icon.height, icon.width? Or if for some reason you wanted it that nested, icon.size.height and icon.size.width.

Reply:
changed li to div and adding width and height to data object.